src/server.ts

```diff
@@ -0,0 +1,29 @@
+import express, { Express, NextFunction, Request } from 'express'
+import { setup, serve } from 'swagger-ui-express'
+
+import cors from 'cors'
+import { json } from 'body-parser'
+import errorHandler from './lib/error-handler'
+
+import { RegisterRoutes } from './routes'
+import * as swaggerJson from './swagger.json'
+import Database from './lib/db'
+import logger from './lib/logger'
```

Comment:
The folder is called `lib/Logger` can you be lowercase the folder name for consistency.

Reply:
yes, it should have been lower :facepalm: ,resolved